docs/data.md

```diff
@@ -1,40 +1,60 @@
 
-## Gridded model reanalysis data
-Reanalysis data is useful for providing a very long history of atmospheric conditions. The ERA5 dataset is a well known and widely used global reanalysis dataset.
+### Working with NetCDF Data
 
-For more information see https://www.ecmwf.int/en/forecasts/datasets/reanalysis-datasets/era5 and
-https://github.com/pangeo-data/WeatherBench
+To use `scores` with [NetCDF](https://doi.org/10.5065/D6H70CW6) or [HDF5](https://doi.org/10.11578/dc.20180330.1) data, install [h5netcdf](https://github.com/h5netcdf/h5netcdf). The h5netcdf library is included in the `scores` ["all"](installation.md#all-dependencies-excludes-some-maintainer-only-packages) and ["tutorial"](installation.md#tutorial-dependencies) installation options. Opening NetCDF data is demonstrated in [this tutorial](project:./tutorials/First_Data_Fetching.md). 
 
-## Gridded satellite (observation) data
-Satellite data varies according to the type of orbit and purpose of the mission. It is too complex to quickly address in a demonstration. A guide on working with satellite data may be added in future.
+## Weather and Climate Data
 
-## Gridded radar (observation) data
-Radar data also varies according to region and is not a globally standardised data set. Information on Australian based radars can be found at https://www.openradar.io/
+This section provides a brief overview of some commonly used weather and climate datasets, and a software package for accessing meteorological and climatological data.
 
-## Working with GRIB data
-To use `scores` with GRIB data, install [cfgrib](https://github.com/ecmwf/cfgrib) and use `engine='cfgrib'` when opening a grib file with `xarray`.
+### Datasets
 
-## Working with NetCDF data
-To use `scores` with NetCDF or HDF5 data, install [h5netcdf](https://github.com/h5netcdf/h5netcdf). Opening NetCDF data is demonstrated in the notebook tutorials and the `h5netcdf` library is included in the tutorial dependencies.
+#### Gridded Global Numerical Weather Prediction Data
+
+Global numerical weather prediction (NWP) models are used to generate medium range forecasts and provide the initial and boundary conditions for higher-resolution regional models. Their global coverage makes them a good starting point for demonstrating the application of scoring methods in any region of interest.
+
+Archived datasets are available for:
+
+- Bureau of Meteorology's Australian Parallel Suite version 3 (APS3) Australian Community Climate and Earth-System Simulator (ACCESS), see [https://doi.org/10.25914/608a993391647](https://doi.org/10.25914/608a993391647).
+- National Oceanic and Atmospheric Administration (NOAA) Global Forecast System (GFS), see [https://www.ncei.noaa.gov/products/weather-climate-models/global-forecast](https://www.ncei.noaa.gov/products/weather-climate-models/global-forecast).
+
+#### Point-Based Data
+
+Point-based observations are shared routinely between countries for the purposes of weather modelling.
+
```

Comment:
I wasn't sure whether this sentence should be retained, and if so, I wasn't sure where in the section it should go.

Reply:
This sentence has since been modified, so this is now resolved.